rts/Rml/SolLua/README.md

```diff
@@ -0,0 +1,65 @@
+RmlSolLua
```

Comment:
I see, so the whole rts/Rml/SolLua/ folder is vendored by copy-pasting and editing, not using something like git submodules? That feels icky to me, to be honest. But I do not understand the code sufficiently in-depth to know if there are any better solutions for this case.

Reply:
Could one fork https://github.com/LoneBoco/RmlSolLua on GitHub, edit that fork, and then use git submodule on that?

Reply:
Replying to https://github.com/beyond-all-reason/spring/pull/1415#discussion_r1554658441 .

I do not know how to best handle the inclusion of a BAR-specific change, it would have been nice if there was a way to avoid including BAR-specific changes in that source code itself, have the BAR-specific changes in a separate folder, and then keep the fork of the library as a git submodule. While the original library has not been updated, there is a fork in https://github.com/nimble0/RmlSolLua , with some changes 6 months ago. I do not know if those changes are relevant, however.

@ChrisFloofyKitsune Thank you for your answer, I consider this comment of mine to have been resolved by you, please do mark it resolved as you see fit.